### PR TITLE
remove extraneous args param from init

### PR DESCRIPTION
**Pull request by jramapuram**
- Filelogger had an extra args param that caused a crash in `train-pytorch.py`
  - fix return value in test fn

### CODE REVIEW

File: examples/plot_results.py [hunk 1]
```diff
@@ -18,11 +18,11 @@
 
 acc = np.asarray(acc)
 x = [i*50000/1e6 for i in range(1, len(acc))]
-y = [np.amax(acc[:i])/100.0 for i in range(1, len(acc))]
+y = [np.amax(acc[:i]) for i in range(1, len(acc) + 1)]
```
Comment (rwebb) [inline examples/plot_results.py:21]: +1 crashes for me because x and y are not the same length.

File: train-pytorch.py [hunk 2]
```diff
@@ -174,6 +174,8 @@ def test(epoch, model, data_loader, logger, loss_function=nnf.cross_entropy):
         logger.record('test', 'acc', correct_percent / 100.0)
         logger.new_row('test')
 
+    return correct_percent
```
Comment (rwebb) [inline train-pytorch.py:177]: good catch

File: train-pytorch.py [hunk 3]
```diff
@@ -235,7 +235,6 @@ def run():
                         ],
                         path=args.filelog,
                         file_prefix=short_name,
-                        args=args
```
Comment (rwebb) [inline train-pytorch.py:238]: I wanted this so people can customize the file logger and pass argument to it.

File: examples/plot_results.py (hunk 1 above)
Comment (jramapuram) [reply examples/plot_results.py:21]: Will fix to ensure consistency of both.

File: train-pytorch.py (hunk 3 above)
Comment (jramapuram) [reply train-pytorch.py:238]: Makes sense, however the constructor of `FileLogger` doesn't accept an args (or *args) keyword.

**Comment by rwebb** [comment]
I added the missing return.  Think the others are OK.  +1 makes x and y different lengths in the lot.  The args were passed so people can customize filelogger with args.

**Pull request closed without merging**